### DISCUSSION TITLE
Add Vulkan to MMJ build

### DISCUSSION THREAD

**Discussion by Hisham735** [weihuoya/citra#735] [Ideas]
Vulkan is still experimental but can run many games better than the current OpenGL, Mali got huge boost from it, Would be great to see Vulkan in the MMJ build!

**Comment by Tappy-tech** [comment]
When Weihuoya will have energy to add it, they will. Don't expect to come out soon just because they started updating more, and, presumably, got energy.

**Comment by Azazelivae** [comment]
I'm not saying it will because i'm not Weihouya but generally Citra-MMJ in its basis were practically a Lightweight Citra designed for Older-Gen.

So yeah Wei probably gonna add it but then again you said it yourself it is still "Experimental" hell I don't think it was even added to Canary yet then again you can just Build it yourself but that's different matter entirely.

**Comment by boygo2239** [comment]
Citra Vulkan still has black errors in shadow rendering,
The shadow of Citra mmj is fixed! And the performance is stable.
![Screenshot_20230406_212128](https://user-images.githubusercontent.com/54440966/230417189-ddac5638-5387-4b0c-b6e9-f33faa71de6e.jpg)
